linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -145,14 +151,18 @@ static void threadpool_dispose(THREADPOOL* threadpool)
 
 **SRS_THREADPOOL_LINUX_07_015: [** `threadpool_destroy` shall destroy the SRW lock by calling `srw_lock_destroy`. **]**
 
+**S_R_S_THREADPOOL_LINUX_07_089: [** `threadpool_close` shall signal all the threads that `threadpool` is closing by calling `InterlockedHL_SetAndWakeAll`.  **]**
```

Comment:
>threadpool_close [](http://example.com/codeflow?start=38&length=16)

This can't be close, it's probably meant to be threadpool_dispose

Reply:
Corrected